python/ikarus/CMakeLists.txt

```diff
@@ -12,9 +12,10 @@ dune_add_pybind11_module(NAME _ikarus)
 
 set_property(
   TARGET _ikarus
-  PROPERTY LINK_LIBRARIES dunecommon ikarus _io
+  PROPERTY LINK_LIBRARIES dunecommon ikarus
   APPEND
 )
+add_dependencies(_ikarus _io)
```

Comment:
Is this needed? I wondered if we really need it since I saw that i also registered the material  tags differently and less complicated. You also change in another PR the ENUM Macro that it can receive the module. then we can get rid of _io.cc again, which caused headaches. But this should happen in the other PR.

Reply:
It is actually needed, otherwise the _io doesn't get compiled when using Ctest

But yeah i saw that when working with the materials as well, I ll change that in another PR